frontend/src/pages/Section.tsx

```diff
@@ -89,12 +93,29 @@ const Section: FC = () => {
         ))}
         <Chip
           sx={{ '& .MuiChip-label': { paddingX: 0.5, display: 'flex' } }}
+          onClick={() => {
+            setOpenPop(true);
+          }}
           label={
             <SvgIcon htmlColor={textSecondaryColor} width="24px">
               <PlusIcon />
             </SvgIcon>
           }
         />
+        <TextPopup
+          isOpen={isTextPopupeOpen}
+          parentSection={`${t(StringBank.SECTION)} ${section?.index}`}
+          newVersionName={
+            section && section.versions
+              ? `${t(StringBank.VERSION)} ${section!.versions!.length + 1}`
```

Comment:
Let's remove assertions. We may use optional chaining instead

Reply:
done, moved to a function